### PR TITLE
Update house-style.md

### PR DESCRIPTION
**Pull request by o-t-w**

### CODE REVIEW

File: javascript/house-style.md [hunk 1]
```diff
@@ -998,7 +998,7 @@ function init(settings) {
     const selectors = Object.assign({}, defaults, settings)
 }
 
-module.exports = {init};
+module.export = {init};
```
[the remark this answers is not in the document]
Comment (o-t-w) [reply javascript/house-style.md]: I'm not come across `module.export` before so I have no idea 🤷‍♀️

[the remark this answers is not in the document]
Comment (o-t-w) [reply javascript/house-style.md]: Is `module.exports` a CommonJS thing? I'm don't think its at all valid in native module syntax.

[the remark this answers is not in the document]
Comment (davidpauljunior) [reply javascript/house-style.md]: It's a node thing.  
Presumably this should be updated the syntax used in the examples above?

[the remark this answers is not in the document]
Comment (o-t-w) [reply javascript/house-style.md]: I made another PR

**Pull request merged**